proto/README.md

```diff
@@ -4,6 +4,5 @@
 2. Rebuild the generated Go code
 
 ```sh
-$ cd v1alpha2
-$ go generate ./proto/
```

Comment:
The problem seems to be related to wrong directory name. If you `cd` to `v1alpha2`, there is no `proto` directory there. Your fix is OK, another option is to run `go generate ./v1alpha2/` from the `proto` directory.

Reply:
We will also need to generate `pipeline` directory. So I wrote `go generate ./v1alpha2/`

Reply:
I'm a bit confused. The change suggest running `go generate -v ./...`.
I think it also helps specifying which is the directory the command in run from. If called from the repo root, more changes are generated compared to when running from {root}/proto.

Reply:
Sorry, I meant I wrote `go generate -v ./...` because we want to run generate for both directories.  Since this Readme is in the proto directory, it implies running from this directory.

Reply:
OK.
/lgtm